docs/content/any/project/changelogs/NEXT.md

```diff
@@ -48,5 +48,6 @@ Link to [relevant documentation section]().
 #### Other bugs & improvements
 
 - The debugger can now break on rule matches.
+- Reserved words can be used as field and method names in dictionaries and objects.
```

Comment:
Maybe give some examples of what reserved words are — I don't think we use that term anywhere else in the docs. Maybe "Reserved Polar keywords" would be more descriptive?

Reply:
👍  done